02-household-queries/question_answer_citations.json

```diff
@@ -0,0 +1,169 @@
+[
+    {
+        "id": 0,
+        "orig_question": "",
+        "question": "",
+        "orig_answer": "",
+        "answer": "",
+        "guru_cards": [
+            ""
+        ]
+    },
+    {
+        "id": 1,
+        "orig_question": "The client has her 19 year old grandchild living with her. She says she does her own thing and they don't really share food. Does the grandchild need to be included?",
+        "question": "The client has her 19 year old grandchild living with her but they don't really share food. Does the client need to include the grandchild on the application?",
+        "orig_answer": "No. grandchildren are not mandatory household members",
+        "answer": "No",
+        "guru_cards": [
+            "Who are mandatory HH members for food stamps?"
+        ]
+    },
+    {
+        "id": 2,
+        "orig_question": "The client's son is 20 and is still living with them, but has his own job and buys his own food. The client doesn't want to list him on the application, do they have to? If so, do we need to include the son's income?",
+        "question": "The client's son is 20, is still living with them, but has his own job and buys his own food. Does the client have to list him on the application? If so, do we need to include the son's income?",
+        "orig_answer": "Yes, since the son is under 22, he is a mandatory household member and needs to be on the application. His income would also need to be listed since he is no longer a minor.",
+        "answer": "Yes",
+        "guru_cards": [
+            "Who are mandatory HH members for food stamps?",
+            "How is the income of a dependent counted for SNAP?"
+        ]
```

Comment:
@ccheng26 Use `guru_cards` to evaluate if the desired cards are being retrieved for the scenarios.